app/helpers/application_helper/button/host_feature_button.rb

```diff
@@ -2,7 +2,19 @@ class ApplicationHelper::Button::HostFeatureButton < ApplicationHelper::Button::
   needs :@record
 
   def visible?
-    return false if @record.kind_of?(ManageIQ::Providers::Openstack::InfraManager)
+    if @record.kind_of?(ManageIQ::Providers::Openstack::InfraManager)
+      return true if ['start', 'stop'].include?(@feature.to_s)
+      return false
+    end
+
+    if @record.kind_of?(ManageIQ::Providers::Openstack::InfraManager::Host)
+      case @feature.to_s
+        when 'start'
+          return @record.validate_start[:available]
```

Comment:
@rwsu You don't need to call the validate methods this way, that's what [`AvailabilityMixin`](https://github.com/rwsu/manageiq/blob/da5cddef6509e5d54457ca2dd7885f088ca84265/app/models/mixins/availability_mixin.rb#L23-L24) was for. So using 
```ruby
@record.is_available?(:start)
```
was the correct way, if you'd want to use `AvailabilityMixin`, but the thing is, **we [don't want to use `AvailabilityMixin`](https://github.com/rwsu/manageiq/blob/da5cddef6509e5d54457ca2dd7885f088ca84265/app/models/mixins/availability_mixin.rb#L4) anymore**.

So you **should not** use validate methods in here:
- `app/models/manageiq/providers/openstack/infra_manager.rb`
- `app/models/manageiq/providers/openstack/infra_manager/host.rb`

You **should** use the new `SupportsFeatureMixin` for this *(usage is nicely documented [here](https://github.com/rwsu/manageiq/blob/da5cddef6509e5d54457ca2dd7885f088ca84265/app/models/mixins/supports_feature_mixin.rb#L3-L57))* :wink: